Kitodo/src/test/resources/metadata/2/jpgs/max/.gitignore

```diff
@@ -0,0 +1 @@
+*
```

Comment:
please remove after test instead of ignoring it.

Reply:
This is to commit an „empty folder” to git.